assets/src/functions/hydrateBlock.js

```diff
@@ -1,8 +1,10 @@
 export const hydrateBlock = (blockName, Component) => {
-  document.querySelectorAll( `[data-hydrate="${blockName}"]` ).forEach(
+  const blocks = document.querySelectorAll( `[data-hydrate="${blockName}"]` );
+  console.log(blocks);
+  blocks.forEach(
     blockNode => {
       const attributes = JSON.parse( blockNode.dataset.attributes );
-      ReactDOM.hydrate( <Component { ...attributes.attributes } />, blockNode );
+      ReactDOM.hydrate( <Component { ...attributes } />, blockNode );
```

Comment:
I removed the extra `attributes` level here. In the code to render front end components it was only still using it because it's hard to change because of block content validation in the editor.